ref_status = 'Pending'

if ref_status != 'Successful':
    raise Exception("Sorry, no numbers below zero")
